sdks/python/apache_beam/yaml/standard_providers.yaml

```
@@ -75,6 +76,10 @@
         drop: 'drop'
         fields: 'fields'
         error_handling: 'errorHandling'
+      'Filter-generic':
+        language: 'language'
+        keep: 'keep'
+        error_handling: 'errorHandling'
```

```suggestion
        error_handling: 'error_handling'
```
#31374 is merged, so SchemaTransform config names should be snake_case